.gitlab/binary_build/otel_agent.yml

```diff
@@ -9,6 +9,9 @@
     - !reference [.retrieve_linux_go_deps]
     - inv -e otel-agent.build
   needs: ["go_deps"]
+  variables:
+    KUBERNETES_MEMORY_REQUEST: "8Gi"
+    KUBERNETES_MEMORY_LIMIT: "8Gi"
```

Comment:
Just noting this is only 2Gi higher than the default, may need more.

Reply:
Seems like this is at least getting the CI jobs to pass, but we'll keep an eye on it!